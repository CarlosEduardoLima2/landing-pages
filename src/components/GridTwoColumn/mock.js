export default {
  title: 'GridTwoColum',
  text: `
      Lorem ipsum dolor sit amet consectetur adipisicing elit.
      Iure unde ad velit tempore odit obcaecati,
      vel voluptatem repudiandae similique magni
      rerum architecto maiores a doloremque, ex,
      sapiente aliquid nemo quia.
    `,
  srcImg: 'assets/images/javascript.svg',
};
