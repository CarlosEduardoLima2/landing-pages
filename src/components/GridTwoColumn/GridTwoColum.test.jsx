import { GridTwoColum } from '.';
import { renderTheme } from '../../styles/render-theme';

import mock from './mock';

describe('<GridTwoColum />', () => {
  it('should render grid with two columns', () => {
    const { container } = renderTheme(<GridTwoColum {...mock} />);
    expect(container).toMatchSnapshot();
  });
});
