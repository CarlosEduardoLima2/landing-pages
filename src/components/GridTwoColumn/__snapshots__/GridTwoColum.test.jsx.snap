// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridTwoColum /> should render 1`] = `
.c5 {
  color: #0A1128;
  font-size: 6.4rem;
  text-transform: uppercase;
}

.c2 {
  display: grid;
  grid-template-columns: 1fr 2fr;
  align-items: center;
  gap: 3.2rem;
}

.c2 .c4 {
  margin-bottom: 4.0rem;
}

.c0 {
  background: #fff;
  color: #0A1128;
  min-height: 100vh;
  display: flex;
  align-items: center;
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
}

.c6 {
  font-size: 2.4rem;
}

@media (max-width: 768px) {
  .c5 {
    font-size: 4.0rem;
  }
}

@media (max-width: 768px) {
  .c2 {
    grid-template-columns: 1fr;
    text-align: center;
  }
}

@media (max-width: 768px) {
  .c3 {
    margin-bottom: 3.2rem;
  }
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class="c3"
        >
          <h1
            class="c4 c5"
          >
            GridTwoColum
          </h1>
          <p
            class="c6"
          >
            
      Lorem ipsum dolor sit amet consectetur adipisicing elit.
      Iure unde ad velit tempore odit obcaecati,
      vel voluptatem repudiandae similique magni
      rerum architecto maiores a doloremque, ex,
      sapiente aliquid nemo quia.
    
          </p>
        </div>
        <div
          class=""
        >
          <img
            alt="GridTwoColum"
            class=""
            src="assets/images/javascript.svg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<GridTwoColum /> should render grid with two columns 1`] = `
.c5 {
  color: #0A1128;
  font-size: 6.4rem;
  text-transform: uppercase;
}

.c2 {
  display: grid;
  grid-template-columns: 1fr 2fr;
  align-items: center;
  gap: 3.2rem;
}

.c2 .c4 {
  margin-bottom: 4.0rem;
}

.c0 {
  background: #fff;
  color: #0A1128;
  min-height: 100vh;
  display: flex;
  align-items: center;
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
}

.c6 {
  font-size: 2.4rem;
}

@media (max-width: 768px) {
  .c5 {
    font-size: 4.0rem;
  }
}

@media (max-width: 768px) {
  .c2 {
    grid-template-columns: 1fr;
    text-align: center;
  }
}

@media (max-width: 768px) {
  .c3 {
    margin-bottom: 3.2rem;
  }
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class="c3"
        >
          <h1
            class="c4 c5"
          >
            GridTwoColum
          </h1>
          <p
            class="c6"
          >
            
      Lorem ipsum dolor sit amet consectetur adipisicing elit.
      Iure unde ad velit tempore odit obcaecati,
      vel voluptatem repudiandae similique magni
      rerum architecto maiores a doloremque, ex,
      sapiente aliquid nemo quia.
    
          </p>
        </div>
        <div
          class=""
        >
          <img
            alt="GridTwoColum"
            class=""
            src="assets/images/javascript.svg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
